tui-scrollview/src/scroll_view.rs

```diff
@@ -787,10 +845,10 @@ mod tests {
         let mut buf = Buffer::empty(Rect::new(0, 0, 7, 5));
         let mut state = ScrollViewState::default();
         let mut list_state = ListState::default();
-        let items: Vec<String> = (1..=10).map(|i| format!("Item {}", i)).collect();
+        let items: Vec<String> = (1..10).map(|i| format!("Item {}", i)).collect();
```

Comment:
This was intentional. I wanted 10 items, not 9 in the list. What's the purpose of this change?

Reply:
Whoops, that wasn't supposed to be in the commit. Change dropped.